src/ui/components/VPNSubscriptionOption.qml

```diff
@@ -0,0 +1,186 @@
+        Rectangle {
+            radius: Theme.cornerRadius
+            opacity: checked ? 1 : 0
+            color: checked ? Theme.purple60 : Theme.white
+            width: Theme.windowMargin
+            anchors.left: parent.left
+            anchors.top: parent.top
+            anchors.bottom: parent.bottom
+
+            Rectangle {
+                anchors.right: parent.right
+                anchors.top: parent.top
+                anchors.bottom: parent.bottom
+                width: 4
+                color: Theme.white
+            }
+
+            Behavior on color {
+                ColorAnimation {
+                    duration: 100
+                }
+            }
+        }
+
+        // Purple border when product is selected or focused
+        Rectangle {
+            anchors.fill: parent
+            radius: Theme.cornerRadius
+            border.color: checked || radioDelegate.focus ? Theme.purple60 : Theme.white
+            color: "transparent"
+
+            Behavior on border.color {
+                ColorAnimation {
+                    duration: 100
+                }
+            }
+        }
+    }
+
+    RowLayout {
+        anchors.fill: parent
+        anchors.margins: 12
+        anchors.leftMargin: 24
+        anchors.rightMargin: 24
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.bottomMargin: 12
+
+        ColumnLayout {
+            spacing: 0
+
+            VPNBoldLabel {
+                //TODO: Pull monthly price from model when available
+                property var productMonthlyPrice: "$X.YZ"
+
+                //: “/month” stands for “per month”. %1 is replaced by the monthly cost (including currency).
+                //% "%1/month"
+                text: qsTrId("vpn.subscription.price").arg(productMonthlyPrice)
+
+                lineHeightMode: Text.FixedHeight
+                lineHeight: Theme.labelLineHeight
+                verticalAlignment: Text.AlignTop
+            }
+
+            VPNLightLabel {
+                function getSubscriptionDuration(product) {
+                    switch (product) {
+                    case VPNIAP.ProductMonthly:
+                        return 1;
+                    case VPNIAP.ProductHalfYearly:
+                        return 6;
+                    case VPNIAP.ProductYearly:
+                        return 12;
+                    default:
+                        return 0;
+                    }
+                }
+
+                // TODO (maybe) - Do we want to add the subscription duration in months to the model?
+                property var subscriptionDuration: getSubscriptionDuration(productType)
+
+                //% "Monthly plan"
+                property string productSingleMonth: qsTrId("vpn.subscription.monthlyPlan")
+
+                //: %1 is replaced by the subscription duration in months. %2 is replaced by the total subscription cost.
+                //% "%1-month plan: %2"
```

Comment:
This won't work for most locales that have complex plural forms (imagine having to decline _month_ differently for 3 and 6). I'm going to suggest to use `Multiple months (%1): %2`, but it's far from great, and we should look into a proper fix if possible (i.e. changing the sentence)

@bakulf 
Imagine translating this into Italian, where the form "x-month" doesn't make any sense. It would be "Plan for x months". Now imagine having to translate it for 1 and 1+.